### ISSUE TITLE
Unit tests are not running in Github Actions

### ISSUE THREAD

**Issue by GearsAD** [NavAbility/NavAbilitySDK.py#61]

**Comment by jim-hill-r** [comment]
Agreed unit tests should run on every push. Many are integration tests. Those are also important, just thinking through when they should run. Probably only on merge or PR or even nightly?

**Comment by dehann** [comment]
Testing on PR and merge seems good to me

**Comment by GearsAD** [comment]
The tests don't actually run though in GitHub, somethings missing
